.github/workflows/cron.yml

```diff
@@ -0,0 +1,338 @@
+on:
+  push:
+  schedule:
+    - cron: 0 0 * * * # at the end of each day
+
+jobs:
+  check_for_public_api_diffs:
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: zcash/librustzcash
+          path: librustzcash
+
+      - uses: actions/checkout@v4
+        with:
+          repository: eigerco/uniffi-zcash-lib
+          path: uniffi-zcash-lib
+
+      # caches installed cargo tools
+      - uses: Swatinem/rust-cache@v2
+
+      # Exports:
+      # USED_LIBS - list of librustzcash library names, separated by ';', which are used in the uniffi projects (ex - 'zcash_primitives;zcash_address;')
+      - name: Get librustzcash libraries used in uniffi-zcash-lib
+        run: |
+          {
+            echo 'USED_LIBS<<EOF'
+              # get the names of all librustzcash packages
+              cargo metadata --format-version=1 --no-deps --manifest-path=./librustzcash/Cargo.toml | \
+              jq -r '.packages[] | .name' | \
+              # intersect their names with the dependencies from uniffi-zcash-lib,
+              # in order to get the librustzcash libraries used as dependencies in uniffi-zcash-lib
+              # also remove duplicates
+              xargs -I {} sh -c "cargo metadata --format-version=1 --no-deps --manifest-path=./uniffi-zcash-lib/lib/Cargo.toml | jq -r '.packages[] | .dependencies[] | .name' | grep '{}' | sort -u"
+            echo EOF
+          } >> "$GITHUB_ENV"
+
+        # Exports:
+        # OUTDATED_LIBS - list of library names, separated by ';', which are outdated (ex - 'zcash_primitives;zcash_address;')
+        # ISSUE_LABELS - The labels that are used when searching for or creating an issue. In format 'lib_name-current_ver-latest_ver'
+        # LIBS_UP_TO_DATE - a boolean, indicating whether librustzcash libs we use as deps are up to date
+      - name: Get outdated librustzcash lib versions used in uniffi-zcash-lib
+        env:
+          CARGO_TERM_COLOR: always
+        run: |
+          # TODO: No idea why cargo outdated doesn't work without cd
+          # I works with manifest-path locally, but not here
+          cd ./uniffi-zcash-lib/lib
```

Comment:
This is a weird one... seems like `manifest-path` doesn't do anything and it only happens in CI, iirc. Not sure why.